From 6a64bf71e7a69b3bb5cfafd0d77402967a54c401 Mon Sep 17 00:00:00 2001
From: bb10 <607398+bb10@users.noreply.github.com>
Date: Wed, 16 Jan 2019 05:49:12 +0100
Subject: [PATCH 1/2] Update joystick.xml

Changed B back to the original action of Stop and changed the long press action to Fullscreen.
---
 system/keymaps/joystick.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/system/keymaps/joystick.xml b/system/keymaps/joystick.xml
index c4b50cd4829b..f9f9719fd6dd 100644
--- a/system/keymaps/joystick.xml
+++ b/system/keymaps/joystick.xml
@@ -91,8 +91,8 @@
   <FullscreenVideo>
     <joystick profile="game.controller.default">
       <a>Pause</a>
-      <b>Fullscreen</b>
-      <b holdtime="500">Stop</b>
+      <b>Stop</b>
+      <b holdtime="500">FullScreen</b>
       <x>OSD</x>
       <y>FullScreen</y>
       <start>Info</start>

From 592ed4a3b089a2a852b7deeb45fb81521df1d6a7 Mon Sep 17 00:00:00 2001
From: bb10 <607398+bb10@users.noreply.github.com>
Date: Fri, 25 Jan 2019 21:02:31 +0100
Subject: [PATCH 2/2] Update joystick.xml

Add A and B button functionality to <FullscreenLiveTV> and <FullscreenRadio>
---
 system/keymaps/joystick.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/system/keymaps/joystick.xml b/system/keymaps/joystick.xml
index f9f9719fd6dd..41fd40eea800 100644
--- a/system/keymaps/joystick.xml
+++ b/system/keymaps/joystick.xml
@@ -169,6 +169,9 @@
   </FullscreenGame>
   <FullscreenLiveTV>
     <joystick profile="game.controller.default">
+      <a>Pause</a>
+      <b>Stop</b>
+      <b holdtime="500">FullScreen</b>
       <x>OSD</x>
       <back>OSD</back>
       <guide>OSD</guide>
@@ -186,6 +189,9 @@
   </TVGuide>
   <FullscreenRadio>
     <joystick profile="game.controller.default">
+      <a>Pause</a>
+      <b>Stop</b>
+      <b holdtime="500">FullScreen</b>
       <x>OSD</x>
       <back>OSD</back>
       <guide>OSD</guide>
